parent/pom.xml

```diff
@@ -189,6 +194,12 @@
             </dependency>
             <dependency>
                 <groupId>org.wso2.carbon</groupId>
+                <artifactId>org.wso2.carbon.util.feature</artifactId>
```

Comment:
according to the above kernel feature pom, the Carbon utils bundle is added to it. There is no org.wso2.carbon.util.feature.